src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,30 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+	@Test
+	void split() {
+		String[] result = "1,2".split(",");
+		assertThat(result).contains("1");
+		assertThat(result).contains("2");
+		assertThat(result).containsExactly("1", "2");
+	}
+
+	@Test
+	void substring() {
+		String input = "(1,2)";
+		assertThat(input.substring(1, input.length() - 1)).isEqualTo("1,2");
+	}
+
+	@Test
+	void charAt() {
+		String input = "abc";
+		assertThatThrownBy(() -> {
+			input.charAt(input.length());
+		}).isInstanceOf(StringIndexOutOfBoundsException.class)
+		  .hasMessageContaining("String index out of range: " + input.length());
```

Comment:
```suggestion
		assertThatThrownBy(() -> input.charAt(input.length()))
			.isInstanceOf(StringIndexOutOfBoundsException.class)
		  	.hasMessageContaining("String index out of range: " + input.length());
```

람다식을 사용할 때에는 불필요한 바디를 제거할 수도 있습니다.

Reply:
넵, 감사합니다.